### PR TITLE
Wallet nit 

### PR DESCRIPTION
**Pull request by Jibz-Mysten**
## Description 

Describe the changes or additions included in this PR.

- Update the address input to support the new address size
- added validateOnMount to the stake form 
- Center validatorLogo

## Test Plan 
Screenshot 2023-03-16 at 5 01 01 PM

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/wallet/src/ui/app/components/address-input/index.tsx [hunk 1]
```diff
@@ -49,33 +49,35 @@ export function AddressInput({
...
-                    onBlur={field.onBlur}
-                    className={cx(
-                        'w-full text-bodySmall leading-100 font-medium font-mono bg-white placeholder:text-steel-dark placeholder:font-normal placeholder:font-mono border-none resize-none',
-                        meta.touched && meta.error
-                            ? 'text-issue'
-                            : 'text-gray-90'
-                    )}
-                    name={name}
-                />
+                <div className="min-h-[42px] w-full flex items-center pl-3 py-1 pr-0 ">
```
Comment (williamrobertson13) [inline apps/wallet/src/ui/app/components/address-input/index.tsx:58]:
```suggestion
                <div className="min-h-[42px] w-full flex items-center pl-3 py-1 pr-0">
```

super nit haha

**Review by williamrobertson13 — APPROVED** [review]
lgtm

File: apps/wallet/src/ui/app/components/address-input/index.tsx (hunk 1 above)
Comment (mamos-mysten) [reply apps/wallet/src/ui/app/components/address-input/index.tsx:58]: I think we can just remove the `pr-0` class

File: apps/wallet/src/ui/app/staking/validators/ValidatorLogo.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ export function ValidatorLogo({
     return validatorMeta ? (
         <div
             className={cl(
-                ['w-full flex justify-start  font-semibold'],
+                ['w-full flex justify-start font-semibold'],
```
Comment (mamos-mysten) [inline apps/wallet/src/ui/app/staking/validators/ValidatorLogo.tsx:68]: can just use a string here

**Pull request merged**